cloudinit/settings.py

```diff
@@ -26,6 +26,7 @@
         'Azure',
         'AltCloud',
         'OVF',
+        'RbxCloud',
```

Comment:
This should be added at the end, so that people somehow relying on the existing order don't see their behaviour modified.